package.json

```diff
@@ -3,11 +3,23 @@
   "version": "2.0.0",
   "description": "Relic Data for Warframe",
   "main": "dist/index.js",
+  "files": [
+    "dist/*",
+    "!dist/*.ts",
+    "!dist/spec/**",
+    "dist/index.d.ts"
```

Comment:
I don't think npm follows definition imports. I'm not familiar with the files syntax so take the suggestion with a grain of salt.
```suggestion
    "dist/**.d.ts"
```

Reply:
What do you mean?

Reply:
But implicitly it does as you require the index.d.ts you are requiring other types.
```ts
import Generator from './Generator';
import VersionManager from './VersionManager';
import { TitaniaRelic, TitaniaRelicLocation, TitaniaRelicReward, TitaniaRelicRewardItem, TitaniaRelicVaultedInfo, TitaniaWFMInfo } from './Types';
export { Generator, VersionManager, TitaniaRelic, TitaniaRelicLocation, TitaniaRelicReward, TitaniaRelicRewardItem, TitaniaRelicVaultedInfo, TitaniaWFMInfo, };

```

Reply:
I imagine npm would include only index.d.ts and not the other .d.ts files leaving an invalid typing for the other ts projects. I presume it wouldn't transpile with tsc if it was being used as a npm package.

Reply:
it would include anything the index.d.ts references, those can't be removed, i'm just explicitly using this one directly. if you wanna test it, you can install locally and run the build

Reply:
I tested it and it's as i described, when installed as a package it won't grab the declarations, the way tsc is declaring the types is by file, so if you include only Index.d.ts, which it's contents i described above, it'll not have the information of the types declared in them. Basically `new Generator() = any`.

Reply:
Also my suggestion fixes it.

Reply:
Basically as we're not using only one .d.ts declaration we need to include all of them to work properly.

Reply:
oh shoot, i thought you changed that one :facepalm: i'll go update that

Reply:
updated with 745a1aca899e6123e58eb2b9836cb2e8335edce3